File: src/src/main/java/com/mewebstudio/translatable/repository/CategoryRepository.java
package com.mewebstudio.translatable.repository;

import com.mewebstudio.springboot.jpa.translatable.JpaTranslatableRepository;
import com.mewebstudio.translatable.entity.Category;
import com.mewebstudio.translatable.entity.CategoryTranslation;

public interface CategoryRepository extends JpaTranslatableRepository<Category, String, CategoryTranslation> {
}

File: src/main/kotlin/com/mewebstudio/translatable/repository/CategoryRepository.kt
package com.mewebstudio.translatable.repository

import com.mewebstudio.springboot.jpa.translatable.kotlin.JpaTranslatableRepository
import com.mewebstudio.translatable.entity.Category
import com.mewebstudio.translatable.entity.CategoryTranslation

interface CategoryRepository : JpaTranslatableRepository<Category, String, CategoryTranslation>
